proto/ibc/applications/fee/v1/fee.proto

```diff
@@ -10,15 +10,13 @@ option go_package = "github.com/cosmos/ibc-go/modules/apps/29-fee/types";
 // See Fee Payment Middleware spec:
 // https://github.com/cosmos/ibc/tree/master/spec/app/ics-029-fee-payment#fee-middleware-contract
 message Fee {
```

Comment:
@AdityaSripal updated Fee structure as discussed. I'm wondering if this is even necessary as we could just have Fee be `Fee sdk.Coins` instead of 3 `sdk.Coin`

Reply:
how would you break it down to what amount is for `recv`, `ack`, `timeout`?